packages/reactor-api/src/server.ts

```diff
@@ -10,12 +10,18 @@ import {
   KnexAnalyticsStore,
   KnexQueryExecutor,
 } from "@powerhousedao/analytics-engine-knex";
+import https from "https";
+import fs from "fs";
 
 type Options = {
   express?: Express;
   port?: number;
   dbPath: string | undefined;
   client?: PGlite | typeof Pool | undefined;
+  ssl?: {
```

Comment:
Should we call it `https` to keep it consistent with Connect and Vite dev server?